index.html

```diff
@@ -31,11 +31,12 @@ <h3>Advanced algorithms plainly explained</h3>
       </nav>
     </header>
     <main>
-      Welcome to The Algorithmist! There will be content here soon.
+      <p>Welcome to The Algorithmist! There will be content here soon.</p>
       <div class="post">
-        <h2>Aho-Corasick: Fast, Simultaneous Multi-String Search</h2>
-        <p>Something something content preview content preview</p>
-        <a href="/algos/ahocorasick">continue</a>
+        <div>
```

Comment:
~I'm not sure I understand the intent of this section?~ I'm reading this in monochrome and couldn't tell what was deleted. I understand now.

Do you not want any intro text for the posts on the main page? I don't feel strongly either way, just want to verify your intent.

Reply:
Apart from the blurb in the <h3> tag, I'm not a fan of intro text in navigation pages. IMO, it usually ends up getting in the way and making it more difficult to navigate the site.

Reply:
I'm fine with that. It's easy to change back if we need to.